comet_sdk_test_netf.bat

```diff
@@ -0,0 +1 @@
+dotnet test CDP4-SDK-NETF.sln --framework net452 --filter="(TestCategory!~WebServicesDependent) & (TestCategory!~AppVeyorExclusion)"
```

Comment:
we can also do net48, not so interested in net452 anymore now that COMET is also on net48